polar-core/src/terms.rs

```diff
@@ -358,6 +358,12 @@ impl Term {
     }
 }
 
+/// Operand chirality.
```

Comment:
I don't know what chirality means.  Can we use a term that is widely understood?

Reply:
Also is there a use for this concept outside of the argument to the partial `compare` function? If not, what about just moving it into the partial module and calling it ArgumentPosition?

Reply:
![no fun](https://media.giphy.com/media/cmm1d47AzjcQaUYIJR/giphy.gif)

(I didn't know what it meant either)

Reply:
😎

Reply:
Moved it into partial module and updated naming a bit